content/terminalCommands.json

```diff
@@ -9,6 +9,62 @@
     "prettyName": "cd SecBot",
     "path": "/root/whyd"
   },
+  "help": {
+    "prettyName": "help",
+    "path": "/root/whyd",
+    "functionName": "helpTerminalFunction"
+  },
+  "voice events": {
+    "prettyName": "voice events",
+    "path": "/root/whyd/SecBot",
+    "functionName": "SecBotVoiceEventsFunction"
+  },
+  "most active day": {
+    "prettyName": "most active day",
+    "path": "/root/whyd/SecBot",
+    "dependencies": ["voice events"],
+    "functionName": "SecBotVoiceEventsMostActiveDayFunction"
+  },
+  "favorite voice channel": {
+    "prettyName": "favorite voice channel",
+    "path": "/root/whyd/SecBot",
+    "dependencies": ["voice events"],
+    "functionName": "SecBotVoiceEventsFavoriteVoiceChannelFunction"
+  },
+  "mutes": {
+    "prettyName": "mutes",
+    "path": "/root/whyd/SecBot",
+    "dependencies": ["voice events"],
+    "functionName": "SecBotVoiceEventsMutesFunction"
+  },
+  "dms": {
+    "prettyName": "DMs",
+    "path": "/root/whyd/SecBot",
+    "functionName": "SecBotDMsFunction"
+  },
+  "scores": {
+    "prettyName": "Scores",
+    "path": "/root/whyd/SecBot",
+    "functionName": "SecBotScores"
+  },
+  "score highs": {
```

Comment:
are the `id`s of any of these values ever used? e.g. `"score highs"`

Reply:
also, are these sorted in any way? or is it just ad-hoc?

Reply:
i don't think it necessarily needs to change, my eye was simply caught by the fact that there are 3 unique keys for each element here